tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -2434,21 +2413,15 @@ Status StorageManager::group_metadata_consolidate(
       group_name, EncryptionType::NO_ENCRYPTION, nullptr, 0);
 }
 
-Status StorageManager::group_metadata_vacuum(
-    const char* group_name, const Config* config) {
+Status StorageManagerCanonical::group_metadata_vacuum(
+    const char* group_name, const Config& config) {
```

Comment:
again, lost (unused?) functionality concern?

Reply:
IIUC this is addressed by passing in the storage manager's Config one level higher rather than the redundancy here.